internal/abci/abci.go

```diff
@@ -244,7 +244,14 @@ func NewAbciApp(ctx context.Context, cfg *AbciConfig, snapshotter SnapshotModule
 		app.consensusParams.Votes.MaxVotesPerTx = networkParams.MaxVotesPerTx
 	}
 
-	networkParams.InMigration = app.migrator.InMigration(height)
+	migrationStatus := app.migrator.MigrationStatus()
+	fmt.Println("Migration status: ", migrationStatus.String())
```

Comment:
Logger or rm.  I guess this was for debugging, but it's actually good info to print at Info level.